src/components/View/commons/PageHeader/PageHeaderEventDates.jsx

```diff
@@ -49,7 +49,7 @@ const PageHeaderEventDates = ({ content, moment, rrule }) => {
     }
   }
   return content['@type'] === 'Event' ? (
-    <h4 className="py-2">
+    <h2 className="py-2 h4">
```

Comment:
https://iosanita-staging.c3p0.redturtle.it/novita/comunicati/cento-alla-pandurera-lo-spettacolo-per-prevenire-la-dipendenza-da-gioco-dazzardo#:~:text=da%20gioco%20d%27azzardo-,13%2D10%2D2023,-Si%20tiene%20venerd%C3%AC